molgenis-data-rest/src/main/java/org/molgenis/data/rest/exception/IdentifierAndValueException.java

```diff
@@ -0,0 +1,29 @@
+package org.molgenis.data.rest.exception;
+
+import static org.molgenis.data.i18n.LanguageServiceHolder.getLanguageService;
+
+public class IdentifierAndValueException extends RestApiException
```

Comment:
Add Javadoc for the exceptions and their fields to encourage consistent use.
(this comment times n)